src/server/chromium/chromium.ts

```diff
@@ -221,6 +221,7 @@ const DEFAULT_ARGS = [
   '--use-mock-keychain',
   // See https://chromium-review.googlesource.com/c/chromium/src/+/2436773
   '--no-service-autorun',
+  '--enable-use-zoom-for-dsf=false',
```

Comment:
Let's only do this on Mac? I am afraid that screenshots on Windows will change if you disable this feature.

Reply:
All tests pass? I'm afraid that dragging will break on windows if its enabled!

Reply:
I don't have access to a high dpi windows computer, but I wasn't able to repro this issue on linux. Added the flag for macos only for now.